packages/teams-js/src/private/copilot.ts

```diff
@@ -41,12 +48,49 @@ export namespace copilot {
      *
      * @throws Error if {@linkcode app.initialize} has not successfully completed
      */
-    export function getEligibilityInfo(): AppEligibilityInformation {
+    export async function getEligibilityInfo(): Promise<AppEligibilityInformation> {
       ensureInitialized(runtime);
       if (!isSupported()) {
-        throw errorNotSupportedOnPlatform;
+        throw new Error(`Error code: ${errorNotSupportedOnPlatform.errorCode}, message: Not supported on platform`);
```

Comment:
Why has this error changed?

Reply:
I am trying to follow best practices and throwing an Error means that the app developer will get the same information but also a call stack, which is more helpful when debugging.